services/navigation/builtin/builtin.go

```diff
@@ -53,10 +65,17 @@ type Config struct {
 	BaseName           string                 `json:"base"`
 	MovementSensorName string                 `json:"movement_sensor"`
 	MotionServiceName  string                 `json:"motion_service"`
+	VisionServices     []string               `json:"vision_services"`
+
 	// DegPerSec and MetersPerSec are targets and not hard limits on speed
-	DegPerSec    float64                          `json:"degs_per_sec"`
-	MetersPerSec float64                          `json:"meters_per_sec"`
-	Obstacles    []*spatialmath.GeoObstacleConfig `json:"obstacles,omitempty"`
+	DegPerSec    float64 `json:"degs_per_sec,omitempty"`
+	MetersPerSec float64 `json:"meters_per_sec,omitempty"`
+
+	Obstacles                []*spatialmath.GeoObstacleConfig `json:"obstacles,omitempty"`
+	PositionPollingFrequency float64                          `json:"position_polling_frequency,omitempty"`
```

Comment:
These will need to be updated to reflect what is currently in the scope

Reply:
forgot to push up my changes :/ sorry